ix/chains/fixtures/node_types.json

```diff
@@ -248,6 +248,70 @@
     "config_schema": {}
   }
 },
+{
```

Comment:
please pull in `master` and remove this node_type definition. `LLMToolChain` was deprecated in #150